cdk/apps/dev_worker/setup.py

```diff
@@ -0,0 +1,45 @@
+import setuptools
+
+
+with open("README.md") as fp:
+    long_description = fp.read()
+
+
+setuptools.setup(
+    name="dev_worker",
+    version="0.0.1",
+
+    description="An empty CDK Python app",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+
+    author="author",
+
+    package_dir={"": "dev_worker"},
+    packages=setuptools.find_packages(where="dev_worker"),
+
+    install_requires=[
+        "aws-cdk.core==1.26.0",
```

Comment:
You are not listing the dependencies here?

Reply:
Resolved